pkg/model/firewall.go

```diff
@@ -244,14 +244,6 @@ func (b *FirewallModelBuilder) applyNodeToMasterBlockSpecificPorts(c *fi.ModelBu
 	udpRanges := []portRange{{From: 1, To: 65535}}
 	protocols := []Protocol{}
 
-	if b.Cluster.Spec.Networking.Calico != nil {
-		// Calico needs to access etcd
-		// TODO: Remove, replace with etcd in calico manifest
-		klog.Warningf("Opening etcd port on masters for access from the nodes, for calico.  This is unsafe in untrusted environments.")
-		tcpBlocked[4001] = false
-		protocols = append(protocols, ProtocolIPIP)
```

Comment:
at least in OpenStack this is still needed in 1.14. If we remove IPIP protocol, the overlay network does not work properly between nodes

Reply:
Specifically the protocol part? Not the port right? I can add that back no problem.  This just worked just fine on master on aws at least, haven't tried backporting. Thanks!

Reply:
I mean the protocol part

Reply:
Yep, calico -> etcd directly.  Great, added back.